src/main/java/org/cyclops/integrateddynamics/capability/network/NetworkCapabilityConstructors.java

```diff
@@ -21,12 +23,13 @@ public void onNetworkLoad(AttachCapabilitiesEventNetwork event) {
         INetwork network = event.getNetwork();
         PartNetwork partNetwork = new PartNetwork();
         EnergyNetwork energyNetwork = new EnergyNetwork();
+        IEnergyStorage energyChannel = energyNetwork.getChannel(IChanneledNetwork.DEFAULT_CHANNEL);
         event.addCapability(new ResourceLocation(Reference.MOD_ID, "partNetwork"),
                 new DefaultCapabilityProvider<>(() -> PartNetworkConfig.CAPABILITY, partNetwork));
         event.addCapability(new ResourceLocation(Reference.MOD_ID, "energyNetwork"),
                 new DefaultCapabilityProvider<>(() -> EnergyNetworkConfig.CAPABILITY, energyNetwork));
         event.addCapability(new ResourceLocation(Reference.MOD_ID, "energyStorageNetwork"),
```

Comment:
As I look at this again, what do we even need this capability for? Can we just remove it?

Reply:
It is indeed not used anywhere ATM, but it makes sense to have this here IMO. As the network can handle items using the standard IH capability. Maybe some other mod in the future may have a use for this. (It doesn't cause any issues to have this here in any case)

Reply:
But wouldn't you need to use our API to even get the network object at all, before you could get its capabilities? And if you are using our API, shouldn't you pass a channel rather than getting the default one implicitly?

Reply:
It can be convenient for mod developers, so I won't be changing it.